### PR TITLE
Lesson 05 assignment complete

### PR DESCRIPTION
**Pull request by ryanfree1994**
I am submitting my lesson 05 assignment and I must say that Grid and I are not friends.

### CODE REVIEW

**Comment by ryanfree1994** [comment]
Note: I used Firefox Developer for the assignment and I updated the JSON file to accommodate.

File: lesson05/assignment/css/_settings.variables.scss [hunk 1]
```diff
@@ -1 +1,11 @@
-// Colors
+/*theme colors*/
+$black: black;
+$white: white;
+$silver: #C0C0C0;
+$red: red;
+$lemonyellow: #FFFF9F;
+
+/*mars colors; source: https://engelsjk.com/posts/contour-art-of-mars-topography/*/
```
Comment (cherimarie) [inline lesson05/assignment/css/_settings.variables.scss:8]: 🚀

File: lesson05/assignment/gulpfile.js [hunk 2]
```diff
@@ -59,7 +59,7 @@ function browserSync() {
             baseDir: './'
         },
         notify: false,
-        browser: "google chrome"
+        browser: "Firefox Developer Edition"
```
Comment (cherimarie) [inline lesson05/assignment/gulpfile.js:62]: 👍

**Comment by cherimarie** [comment]
CSS Grid is challenging. I have to revisit documentation every time I use it. That said, it's a vast improvement from what we used before. 

Your code looks excellent. While there's only a single provided design, I'd like to see this page coded for responsivity. That should be second nature at this point 😄  Having the sidebar and main content area each become 100% width at smaller screen sizes would be enough to solve the problem. 

Screen Shot 2020-10-13 at 9 40 57 AM

**Pull request merged**